asv_bench/benchmarks/utils/data_shapes.py

```diff
@@ -43,6 +43,10 @@
     "big": [[100_000, 1]],
     "small": [[10_000, 1]],
 }
+BINARY_OP_SERIES_DATA_SIZE = {
+    "big": [[[500_000, 1], [1_000_000, 1]]],
+    "small": [[[5_000, 1], [10_000, 1]]],
```

Comment:
As far as I understand these arrays setup a benchmark for a `[5_000, 1].mul([10_000, 1])` shaped Series', if so the question is: don't we find it useful to also benchmark same-shaped Series'? The same-shaped and different-shaped results may vary a lot and for the sake of perf analyzing it could be useful to have both.

Reply:
+1

Reply:
added